mos_tests/cinder/cinder_tests.py

```diff
@@ -0,0 +1,132 @@
+        self.volume_list = []
+        self.snapshot_list = []
+
+    def tearDown(self):
+        for snapshot in self.snapshot_list:
+            common_functions.delete_volume_snapshot(self.cinder, snapshot)
+        self.snapshot_list = []
+        for volume in self.volume_list:
+            common_functions.delete_volume(self.cinder, volume)
+        self.volume_list = []
+
+    def test_543176_CreatingMultipleSnapshots(self):
+        """ This test case checks creation of several snapshot at the same time
+
+            Steps:
+                1. Create a volume
+                2. Create 70 snapshots for it. Wait for creation
+                3. Delete all of them
+                4. Launch creation of 50 snapshot without waiting of deletion
+        """
+
+        # Precondition: change quota limit for snapshots
+        tenant_id = [t.id for t in self.keystone.tenants.list() if
+                     t.name == os.environ.get('OS_TENANT_NAME')][0]
+        quota = self.cinder.quotas.get(tenant_id).snapshots
+        self.cinder.quotas.update(tenant_id, snapshots=200)
+
+        # 1. Create volume
+        image_id = [image.id for image in self.nova.images.list() if
+                    image.name == 'TestVM'][0]
+        volume = common_functions.create_volume(self.cinder, image_id, 60)
+        self.volume_list.append(volume)
+
+        # 2. Creation of 70 snapshots
+        count = 70
+        initial_snapshot_lst = []
+        for num in xrange(count):
+            snapshot_id = self.cinder.volume_snapshots \
+                .create(volume.id, name='1st_creation_{0}'.format(num + 1))
+            initial_snapshot_lst.append(snapshot_id)
+            self.assertTrue(
+                common_functions.check_volume_snapshot_status(self.cinder,
+                                                              snapshot_id,
+                                                              'available',
+                                                              timeout=60))
+        self.snapshot_list.extend(initial_snapshot_lst)
+
+        # 3. Delete all snapshots
+        for snapshot in initial_snapshot_lst:
+            self.cinder.volume_snapshots.delete(snapshot)
+
+        # 4. Launch creation of 50 snapshot without waiting of deletion
+        new_count = 50
+        new_snapshot_lst = []
+        for num in xrange(new_count):
+            snapshot_id = self.cinder.volume_snapshots \
+                .create(volume.id, name='2nd_creation_{0}'.format(num + 1))
+            new_snapshot_lst.append(snapshot_id)
+        self.snapshot_list.extend(new_snapshot_lst)
+
+        timeout = time.time() + (new_count + count) * 10
+        while [s.status for s in self.cinder.volume_snapshots.list() if
+               s in new_snapshot_lst].count('available') != new_count:
+            if time.time() < timeout:
+                time.sleep(10)
+            else:
+                raise AssertionError("At least one snapshot is not available")
```

Comment:
it is not informative message, let's add information about the count of volumes here.